src/selectors/index.js

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+import * as app from './app';
+import { selectedThreadSelectors as selectedThread } from './per-thread';
+import * as profile from './profile';
+import * as urlState from './url-state';
+import * as icons from './icons';
+import * as publish from './publish';
+import * as zippedProfiles from './zipped-profiles';
+
+type Selectors = {
```

Comment:
Flow should infer this is, so it shouldn't need an explicit type. If you need to export the type, then it can be inferred through the `typeof` operator.

```
const selectors = {
  app,
  profile,
  urlState,
  icons,
  publish,
  zippedProfiles,
  selectedThread,
}

// You could export it like this, although I don't think we really need to.
export type Selectors = $ReadOnly<$Exact<typeof selectors>>;

export default selectors;
```